docs/querying/basics.md

```diff
@@ -176,13 +207,15 @@ following units:
 
 * `ms` - milliseconds
 * `s` - seconds
-* `m` - minutes
+* `m` - minutes - assuming a minute always has 60s<sup>1</sup>
```

Comment:
unnecessary

Reply:
isn't the point that there is no account for leap seconds ?

Reply:
Yes, who knows about leap seconds?  There haven't been any since 2016; I understand there will never be any more leap seconds. [Wikipedia](https://en.wikipedia.org/wiki/Leap_second#International_proposals_for_elimination_of_leap_seconds)

We could perhaps put on some 'concepts' page the detailed definition of time that Prometheus is based on; I do not think it belongs on the "querying basics" page.